### DISCUSSION TITLE
Question about the environmental variables

### DISCUSSION THREAD

**Discussion by ovizii** [outline/outline#2984] [Self Hosting] [answered]
The instructions for docker: https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t state:

```
Configuration

Using this sample file as a reference create a docker.env file to contain the environment variables for your installation.
```

and

`Create a docker-compose.yml file, an example configuration with all dependencies dockerized and environment variables kept in docker.env is as follows:`

but the linked sample .env file => https://github.com/outline/outline/blob/main/.env.sample actually says:

```
Copy this
# file to .env or set the variables in your local environment manually.
```

can somebody please clarify what to name the file containing the environmental variables?

I forgot to mention that the sample docker-compose.yml file listed here https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t has plenty of references do a file named `    env_file: ./docker.env`

**Comment by MrMysterius** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/25854814/150690366-2242dd1d-195f-4557-91be-0f3adccbe180.png)

As you can see in the docker compose, it's grabbing the `docker.env`

**Reply by ovizii** [reply]
> ![image](https://user-images.githubusercontent.com/25854814/150690366-2242dd1d-195f-4557-91be-0f3adccbe180.png)
> 
> As you can see in the docker compose, it's grabbing the `docker.env`

thanks, I had just noticed it and edited my original question :-/ so the instructions of the sample .env file are wrong then.

**Reply by tommoor** [reply]
You can name it anything you like or include the variables in your actual environment – however `.env` is actually looked for by default so if you name it that then the `env_file` line can be removed from docker-compose.

I only wrote these new instructions yesterday so it's possible there are more teething troubles
